src/utils.py

```diff
@@ -5,7 +5,7 @@
 import curies
 import pandas as pd
 import yaml
-from sssom import MappingSetDataFrame
+from sssom.util import MappingSetDataFrame
```

Comment:
## The fix
As I [wrote in the issue](https://github.com/monarch-initiative/medgen/issues/23#issuecomment-2065277361), ense, it is strange that `__init__.py` imports MappingSetDataFrame, so importing from `sssom` rather than `sssom.util` should also work, but it wasn't.